docs/FAQ.md

```diff
@@ -0,0 +1,23 @@
+# SDG FAQs
+
+## Where can I find more details about the synthetic data generation algorithm?
+
+The [Large-Scale Alignment for ChatBots](https://arxiv.org/pdf/2403.01081) research paper does an excellent job of explaining the SDG process in section 3.2.
+
+## How do I estimate the amount of samples a given leaf node will produce in the SDG process for training?
+
+For each question and answer pair in the taxonomy leaf node in a skill file: an estimated 30 synthetic samples will be produced in the training dataset.  
+
+For each knowledge leaf node: the formula to estimate the number of produced synthetic samples in the training dataset is:
+
+```text
+(total cumulative size of knowledge documents / max document chunk size) * number of qna pairs in the knowledge file leaf node * 30 synthetic samples per qna pair
```

Comment:
Does this factor in the amount of samples that will be filtered out?

Reply:
@RobotSail this is an excellent point: it does not since that part is non detemenistic to a certain extent: but folks have been looking to be able to get some guidance on some ball park numbers/ answers to general questions around how the taxonomy is processed through SDG

I should call that out as a disclaimer and will do that